tools/sanity_checks.py

```diff
@@ -481,6 +501,9 @@ def check_files(self, subproject: str, patch_path: Path) -> None:
         if not_permitted:
             not_permitted_str = ', '.join([str(f) for f in not_permitted])
             self.fail(f'Not permitted files found: {not_permitted_str}')
+        if unformatted:
+            unformatted_str = ', '.join([str(f) for f in unformatted])
+            self.fail(f'Not formatted files found: {unformatted_str}')
```

Comment:
For contributors who don't know about `meson format`, can we make sure the error message has clear guidance about how to fix the problem?